generic_fractional_dao/src/dao.ts

```diff
@@ -0,0 +1,240 @@
+import * as _ from "lodash";
+import { TezosToolkit } from "@taquito/taquito";
+import {
+  Expr,
+  isMichelsonData,
+  isMichelsonType,
+  MichelsonData,
+  MichelsonType,
+  packDataBytes,
+  Parser,
+} from "@taquito/michel-codec";
+import { address, Contract, nat } from "smart-contracts-common/type-aliases";
+import { compileExpression, LigoEnv } from "smart-contracts-common/ligo";
+import { $log } from "@tsed/logger";
+import {
+  Fa2Transfer,
+  Fa2TransferDestination,
+  isAddOperator,
+  isRemoveOperator,
+  OperatorParam,
+  UpdateOperator,
+} from "smart-contracts-common/fa2-interface";
+
+export interface DaoStorage {
+  voting_threshold: nat;
+  voting_period: nat;
+  vote_count: nat;
+}
+
+export type DaoLambda = { lambdaMichelson: string; lambdaExp: Expr };
+
+export const voteWithPermit = async (
+  dao: Contract,
+  voter: TezosToolkit,
+  lambda: DaoLambda
+) => {
+  // const signature = await signPermitV2(voter, dao, lambda.lambdaExp);
```

Comment:
Does signPermitV2 not work?

Reply:
Yes, it works. It is just a different way to implement data packing